appyx-interactions/common/src/commonMain/kotlin/com/bumble/appyx/interactions/composable/AppyxInteractionsContainer.kt

```diff
@@ -113,11 +113,11 @@ fun <InteractionTarget : Any, ModelState : Any> AppyxInteractionsContainer(
                     )
                 )
             }
-            .onPointerEvent {
+            .then(if (!appyxComponent.isGesturesEnabled) Modifier else Modifier.onPointerEvent {
                 if (it.type == PointerEventType.Release) {
                     appyxComponent.onRelease()
                 }
```

Comment:
Hey @peposdd thanks for the PR! Looks good, one suggestion: a statement without (!) is easier to read 
```suggestion
            .then(if (appyxComponent.isGesturesEnabled) {
                Modifier.onPointerEvent {
                    if (it.type == PointerEventType.Release) {
                        appyxComponent.onRelease()
                } else {
                    Modifier
                }   
```